src/components/avatar/avatar.ts

```diff
@@ -25,9 +25,18 @@ export default class IgcAvatarComponent extends SizableMixin(LitElement) {
   /** @private */
   public static styles = [styles];
 
+  public set src(val: string) {
```

Comment:
What do you think about using the `watch` decorator instead custom get/set?

Reply:
Yes, but then I would have to create a separate method for resetting the error state and decorate it.
This method will not be called anywhere so I would have to make the linter to ignore it, as it is an error otherwise.
I think this a bit clearer in this scenario.

Reply:
If the method is protected the linter would be fine with it. Personally I think a custom get/set looks more complicated and is error-prone since you may forgot to call `requestUpdate`.